src/bot/cogs/player.py

```diff
@@ -8,29 +9,38 @@
 
 
 class Player(Cog):
+    """Cog holds basic `Player` commands such as `join`"""
+
     def __init__(self, bot: ArenaBot):
         self.bot = bot
-        self._api_url = f"http://{API_URL}"
+        self._api_url = API_URL
```

Comment:
Do we even need this property? We can just use the API_URL constant directly, right?